index.html

```diff
@@ -0,0 +1,116 @@
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>lawyers for justice website</title>
+  <!-- <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC" crossorigin="anonymous"> -->
+  <link rel="stylesheet" href="assets/dist/css/styles.css">
+  <link rel="stylesheet" href="assets/dist/css/place.css">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Lato:wght@400;700&display=swap" rel="stylesheet">
+<link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Lato:wght@900&display=swap" rel="stylesheet">
+</head>
+<body>
+  <div class="container-fluid container-default">
+    <header class="container-fluid container-default">
+      <div class="top">
+        <img src="assets\icons\facebookWhite.svg" class="icon-top-1" alt="facebook image">
+        <img src="assets\icons\twitter-white.svg" class="icon-top-2" alt="twitter image">
+        <ul class="list-group-top">
+          <li class=""><a href="index.html" class="list-item">English</a></li>
+          <li class="list-item"><a href="index.html" class="list-item">Mypage</a></li>
+          <li class="list-item"><a href="index.html" class="list-item">Logout</a></li>
+        </ul>
+      </div>
+      <div class="nav-together container-desk">
+      <img src="assets/icons/x-icon.svg" class="X-icon" alt="closing menu">
+      <a href="index.html"><img src="assets/images/act/Logo.png" class="logo-image" alt="logo image"></a>
+      <nav class="nav-bar" id="nav-bar">
+        <ul class="nav-list">
+          <li><a href="about.html" class="nav-item">About</a></li>
+          <li><a href="index.html" class="nav-item">Program</a></li>
+          <li><a href="index.html" class="nav-item">Join</a></li>
+          <li><a href="index.html" class="nav-item">Sponsor</a></li>
+          <li><a href="index.html" class="nav-item">News</a></li>
+        </ul>
+      </nav>
+      <button class="nav-top-button"><a href="about.html">Join the fight!</a></button>
+      </div>
+      <img src="assets/icons/ham-icon.svg" class="ham-icon" alt="menu">
+    </header>
+    <main class="container-fluid container-default homepage">
+      <section class="fade-image">
+        <span class="img"></span>
+    </section>
+    <section class="rest-homepage">
+      <section class="home-text-section">
+        <h1 class="home-text-1">Hello There!</h1>
+        <h2 class="home-text-2">Welcome to lawyers for justice</h2>
+        <p class="paragraph-home-1">Visit out humble website below.<br>We would love to here from you and what we can do for you.Advocating for positive change with people over 80 countries</p>
```

Comment:
- [ ] please this text is in a block form with white borders . kindly follow the design but feel free to personalize the content.
- [ ] make the date bolder and add text under it
yours
![Screenshot (1009)_LI](https://user-images.githubusercontent.com/65298048/234982744-f33857f8-8753-44b4-a5d1-a9b7df739e66.jpg)
main design

![Screenshot (1008)](https://user-images.githubusercontent.com/65298048/234982875-17dbbaf9-3892-4273-a7a2-2469ffc96a8b.png)